CalendarFXView/src/main/java/com/calendarfx/view/DateControl.java

```diff
@@ -1038,6 +1037,67 @@ public String toString() {
         }
     }
 
+    /**
+     * Possible edit operations with an entry.
+     * This enum will be used as parameter of the callback set with {@link DateControl#setEntryEditPolicy}.
+     */
+    public enum EditOperation {
+
+        /**
+         * Checked if any drag/drop operation is allowed
+         */
+        DRAG_AND_DROP,
+
+        /**
+         * Checked if the start of an entry can be changed (e.g. using drag/drop).
+         * for drag and drop you also have to allow {@link #DRAG_AND_DROP}
+         */
+        CHANGE_START,
+
+        /**
+         * Checked if the end of an entry can be changed (e.g. using drag/drop)
+         * for drag and drop you also have to allow {@link #DRAG_AND_DROP}
+         */
+        CHANGE_END,
+
+        /**
+         * Checked if entry can be moved using drag/drop
+         * for drag and drop you also have to allow {@link #DRAG_AND_DROP}
+         */
+        MOVE,
```

Comment:
Isn't this the same as CHANGE_START and CHANGE_END combined? Where is the difference between MOVE and DRAG_AND_DROP?

Reply:
In my use-case I do have entries where the user is allowed to move the entry, but not to change the length. So MOVE does not allow to change just the start OR the end time.